va/va.h

```diff
@@ -4791,6 +4791,56 @@ typedef struct _VAPictureHEVC
  */
 #define VA_PICTURE_HEVC_RPS_LT_CURR             0x00000040
 
+typedef enum{
+    VACopyObjectSurface = 0,
+    VACopyObjectBuffer  = 1,
+} VACopyObjectType;
+
+typedef struct _VACopyObject {
+    VACopyObjectType  obj_type;    // type of object.
+    union
+    {
+        VASurfaceID surface_id;
```

Comment:
These ID types goes as:
```
typedef unsigned int VAGenericID;
```

which might have different size... Can we add some specific size to this union? Like:
```
    union
    {
        VASurfaceID surface_id;
        VABufferID  buffer_id;
        uint64_t reserved;
    } object;
```

Reply:
to enlarge the size to 64bit?  which compiler/platform will lead to different size of unsigned int?

Reply:
not sure, but I am paranoid:)

Reply:
maybe we could consider about the "uint32_t"/"uint64_t" "unsigned long"  for future, such as 3.0? , unsigned long will be changed adaptively with the compiler. uint64_t /uint32_t will be dedicate size.